src/main/kotlin/no/nav/familie/ef/personhendelse/handler/PersonhendelseHandler.kt

```diff
@@ -20,14 +26,25 @@ abstract class PersonhendelseHandler(
 
     abstract val type: PersonhendelseType
 
+    open fun skalOppretteOppgave(personhendelse: Personhendelse) = true
+
+    abstract fun lagOppgaveBeskrivelse(personhendelse: Personhendelse): String
+
     open fun handle(personhendelse: Personhendelse) {
+
+        if (personhendelse.erAnnulleringEllerKorrigering()) {
+            annullerEllerKorrigerOppgave(personhendelse)
+            return
+        }
```

Comment:
Ettersom metoden her er open, så er den også overskreven i dødsfallhandler, som kanskje også burde bli håndtert. Mulig det burde håndteres i en egen PR?

Reply:
Det er litt av ulempen av bruket av abstract her, når jeg la det til så var jeg ikke helt sikker på hvordan dødsfallhandler skulle håndteres. 
Hadde man ikke hatt att Dødsfall skal håndtere flere personer, så hadde det vært mye ryddigere med bare et interface for handlers, og att det var en egen komponent som ikke var abstract som håndterte selve oppgave-håndteringen etc

Reply:
Ja, vi får se, synd at det ikke passer helt